.gdlintrc

```diff
@@ -20,6 +20,7 @@ comparison-with-itself: null
 constant-name: _?[A-Z][A-Z0-9]*(_[A-Z0-9]+)*
 disable: [
   unused-argument,
+  max-line-length,
```

Comment:
Is this required? I would like to keep the line length under control :)

Reply:
It’s already an issue for test function names unfortunately.

Reply:
I see ... in that case, let's disable this :+1: 

I gotta say though, the linter is less than impressive by far. I don't think I've ever had a problem with too long method names in Godot Editor itself. I've checked the test method `test_the_quick_brown_fox_jumps_over_the_lazy_dog` and it works perfectly fine. I thought that this toolkit is going to recreate the behavior our users might have while using Godot Engine, but more and more checks are out of sync :(